clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1687,8 +1687,8 @@ fork_init(void *drcontext)
 DR_EXPORT void
 drmemtrace_client_main(client_id_t id, int argc, const char *argv[])
 {
-    /* We need 2 reg slots beyond drreg's eflags slots => 3 slots */
-    drreg_options_t ops = { sizeof(ops), 3, false };
+    /* drreg will add 1 slot for eflags internally */
+    drreg_options_t ops = { sizeof(ops), 2, false };
```

Comment:
The title says "in tracing client": please be specific as there are many such clients.

Reply:
Done